src/service/referral-api/tests/FamilyHubs.ReferralApi.UnitTests/BaseUnitTest.cs

```diff
@@ -0,0 +1,86 @@
+using AutoMapper;
+using FamilyHubs.Referral.Core;
+using FamilyHubs.ReferralService.Shared.Dto;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+
+namespace FamilyHubs.Referral.UnitTests;
+
+public class BaseUnitTest<T>
```

Comment:
Should this be abstract?

Reply:
Resolved: [7593257](https://github.com/DFE-Digital/fh-services/pull/154/commits/759325774662a684fa86516f6b76feb93e986315)